cli/package.json

```diff
@@ -35,9 +35,9 @@
   "scripts": {
     "postpack": "rm -f oclif.manifest.json",
     "posttest": "tslint -c ../common/tslint.yml -p . -t stylish",
-    "prepack": "rm -rf lib && tsc -b && oclif-dev manifest && oclif-dev readme",
+    "prepack": "rm -rf lib && tsc && oclif-dev manifest && oclif-dev readme",
```

Comment:
Why?

Reply:
Just to keep it inline with the other uses of `tsc`.

Reply:
this line was created by `oclif init`

Reply:
Note that the `-b` is needed for [project references](https://www.typescriptlang.org/docs/handbook/project-references.html) if we decide to use them.

Reply:
As long is it's consistent.